### DISCUSSION TITLE
Updating firmware in the Evnex charger

### DISCUSSION THREAD

**Discussion by smbunn** [lbbrhzn/ocpp#549] [Q&A]
My Evnex supplier has told me I am a year out of date with firmware and sent a link to the latest firmware.
https://s3.ap-southeast-2.amazonaws.com/firmware.evnex.io/ChargePoint_1.2.9.89da3d7_HWV0-1_Release.dfu

He also said that this is easiest updated via a firmware update command in my OCPP installation.  Can this be done in this OCPP implementation and if so, how please?

**Comment by dreimer1986** [comment]
It worked twice on my EVBOX, so... it should. Here what I did:
https://github.com/lbbrhzn/ocpp/issues/221#issuecomment-1035989737
